### ISSUE TITLE
Anki outdated version

### ISSUE THREAD

**Issue by cantalupo555** [flathub/net.ankiweb.Anki#93]
Hello, the package for ARM64 version is very outdated, version 2.1.15
Is there any possibility to get the updated version?

**Comment by ingemarberg** [comment]
I don't think there's any easy fix for this. One possibility might be to use the Python wheels for Anki together with PyQt BaseApp, instead of the binary package we're using right now.

**Comment by cantalupo555** [comment]
> I don't think there's any easy fix for this. One possibility might be to use the Python wheels for Anki together with PyQt BaseApp, instead of the binary package we're using right now.

I already tried using https://betas.ankiweb.net/#via-pypipip.
But the performance is very bad, it crashes all the time.

Now I will try to compile directly from source.

**Comment by federvieh** [comment]
This is strange. I'm pretty sure that in January '23 there was an up-to-date flatpak that worked. I specifically remember, because [I gave up on trying to create a Alpine package](https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/43109#note_283356) because the flatpak version was fine. I wonder what happened.

**Comment by cantalupo555** [comment]
> This is strange. I'm pretty sure that in January '23 there was an up-to-date flatpak that worked. I specifically remember, because [I gave up on trying to create a Alpine package](https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/43109#note_283356) because the flatpak version was fine. I wonder what happened.

Only the ARM64 version has this problem.

**Comment by federvieh** [comment]
> 
> Only the ARM64 version has this problem.

Yes, but I was trying to build it because I wanted to use it on the pinephone (ARM64/aarch64). Therefore, I'm pretty confused now. I wonder if back then maybe the author was distributing flatpaks directly. :thinking:

**Comment by cantalupo555** [comment]
@federvieh https://gitlab.alpinelinux.org/fraolt/aports/-/commit/597085786206e8ffd152f4deba7390f146285e2f

Would a compilation be made for ARM64?
It was mentioned that the compilation became simpler in its Commit.

**Comment by lutzlapalma** [comment]
When Anki starts on a Raspberry Pi, there is this message:

"Your Anki client does not support the new timezone handling yet. Please ensure your Anki is up to date."

After a flatpak upgrade command, this information shows up:

"Info: runtime org.kde.Platform branch 5.12 is end-of-life, with reason:
   We strongly recommend moving to the latest stable version of the Plaform and SDK
Info: applications using this runtime:
   net.ankiweb.Anki

Info: runtime org.freedesktop.Platform.html5-codecs branch 18.08 is end-of-life, with reason:
   The Freedesktop SDK 18.08 runtime is no longer supported as of 31th August 2020. Please ask your application developer to migrate to a supported platform.
Info: applications using this extension:
   net.ankiweb.Anki

Is it really so complicated, to fix this and publish it again on flathub?

Is there an AppImage of Anki?

**Comment by rayes0** [comment]
Duplicate of #125
